openapi-filter.js

```diff
@@ -47,10 +47,23 @@ let argv = require('yargs')
     .number('maxAliasCount')
     .default('maxAliasCount',100)
     .describe('maxAliasCount','maximum YAML aliases allowed')
+    .alias('configFile', 'c')
+    .describe('configFile', 'The file & path for the filter options')
     .help()
     .version()
     .argv;
 
+// apply options from config file if present
+if (argv && argv.configFile) {
+    console.log('Options Config file used: ', argv.configFile);
+    try {
+        let configFileOptions = JSON.parse(fs.readFileSync(argv.configFile, 'utf8'));
+        argv = Object.assign({}, argv, configFileOptions);
```

Comment:
Excellent - being able to set any existing options here was just what I was hoping for here.

Reply:
Made sense, to allow the combination of passing the configFile options and cli parameters.

Reply:
Sorry that I'm late to the party. But I would suggest that you change the order and let the command line arguments (dynamic) take precedence over the config file(static). This way the new file config won't add any breaking changes and it will be easier to override configuration from the config files.

From Object.assign documentation: "Later sources' properties overwrite earlier ones."